composer.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "pd/public-access",
+	"type": "library",
+	"description": "Secured representing claims between two parties",
+	"homepage": "https://github.com/peckadesign/PublicAccessModule",
+	"license": "proprietary",
```

Comment:
když to máme v public repu, tak proprietary asi nebude úplně to pravý vořechový :trollface: